roles/database/tasks/main.yml

```diff
@@ -1,8 +1,12 @@
 ---
 - include: install-apt-deps.yml
-  when: initialize_local_db
+  when: fpsd_database_initialize_local_db
+
 - include: initialize-fpsd-db.yml
-  when: initialize_local_db
+  when: fpsd_database_initialize_local_db
```

Comment:
Is this conditional really necessary? Haven't tested against a remote database, but the included task list looks like it should work idempotently, and return green `OK`s if the relevant tables already exist, without destroying data. Can you elaborate on why it's necessary to skip the task list here?